### PR TITLE
API Docs Complete

### PR DESCRIPTION
**Pull request by roycem90**
Usage documentation still not done

### CODE REVIEW

File: O365/account.py [hunk 1]
```diff
@@ -124,12 +120,12 @@ def schedule(self, *, resource=None):
         return Schedule(parent=self, main_resource=resource)
 
     def storage(self, *, resource=None):
-        """ Get an instance to handle file storage like OneDrive or
-        Sharepoint document libraries for the specified account resource
+        """ Get an instance to handle file storage OneDrive for the
```
[the remark this answers is not in the document]
Comment (roycem90) [reply O365/account.py]: what others?

[the remark this answers is not in the document]
Comment (janscas) [reply O365/account.py]: Sharepoint document libraries and group drives

[the remark this answers is not in the document]
Comment (roycem90) [reply O365/account.py]: we have sharepoint() for that and group drives are also onedrives technically

[the remark this answers is not in the document]
Comment (roycem90) [reply O365/account.py]: any way list anything else u may find, will update them in next commit

[the remark this answers is not in the document]
Comment (janscas) [reply O365/account.py]: I think that's all

**Pull request merged**